edg_core/HierarchyBlock.py

```diff
@@ -338,6 +338,8 @@ def _def_to_proto(self) -> edgir.HierarchyBlock:
         assert not port._get_initializers([port_name]), f"connected boundary port {port_name} has unexpected initializer"
     pb = self._populate_def_proto_port_init(pb)
 
+    pb = self._populate_def_proto_description(pb)
```

Comment:
style nit: should be consistent with the rest of the function - don't need the extra newline